src/charm.py

```diff
@@ -18,6 +22,63 @@ class SpringBootCharm(CharmBase):
     def __init__(self, *args):
         """Initialize the instance."""
         super().__init__(*args)
+        self.framework.observe(self.on.config_changed, self.reconciliation)
+
+    def _spring_boot_container(self) -> ops.model.Container:
+        """Retrieve the container for the Spring Boot application.
+
+        Returns:
+            An instance of :class:`ops.charm.Container` represents the Spring Boot container.
+        """
+        container = self.unit.get_container("spring-boot-app")
```

Comment:
Why not returning it directly instead of assigning it to a variable?

Reply:
Fixed, thanks!